Runtime/Properties/TeleportationProperty.cs

```diff
@@ -102,10 +104,21 @@ protected override void InternalSetLocked(bool lockState)
             }
         }
         
-        protected void EmitTeleported()
+        protected void EmitTeleported(LocomotionSystem locomotionSystem)
```

Comment:
Is this an API break? And do we care?

Reply:
It is, I think we have a couple of releases where we don't care.